docker/suricata/dist/update.yaml

```diff
@@ -0,0 +1,12 @@
+disable-conf: /etc/suricata/disable.conf
+enable-conf: /etc/suricata/enable.conf
+#drop-conf: /etc/suricata/drop.conf
+modify-conf: /etc/suricata/modify.conf
+
+ignore:
+  - "*deleted.rules"
```

Comment:
Per the discussion in #487, I think this file should be ignored. It's also the default suricata-update behavior to ignore it.